stablehlo/dialect/Version.h

```diff
@@ -43,6 +43,10 @@ class Version {
   /// Return a Version representing the minimum supported VHLO dialect version.
   static Version getMinimumVersion() { return Version(0, 9, 0); }
 
+  /// Return the current version for StableHLO APIs.
+  /// Increments on all C++/C/Python API changes
+  static int64_t getApiVersion() { return 1; }
```

Comment:
How about moving this to a free function at the bottom of this file, under the `mlir::stablehlo` namespace? It feels a bit weird to have StableHLO API version under `mlir::vhlo`.

Reply:
Given our recent discussions on VHLO as an implementation detail, would it make sense to move this entire file into `stablehlo` namespace. Then VHLO would be querying the current version of StableHLO.

Reply:
I guess the question is "What is the version source of truth" - Is VHLO targeting a StableHLO version, or is StableHLO's version the VHLO version.

If the former, then probably `stablehlo::Version` makes sense. The latter, `vhlo::Version`. This can be done later. I just find it odd that we expose `stablehlo.get_current_version` via `Serialization.h`, which requests the current VHLO version.